ui/lib/apps/SlowQuery/utils/useSlowQueryTableController.ts

```diff
@@ -179,12 +179,11 @@ export default function useSlowQueryTableController(
         db: queryOptions.schemas,
         digest: queryOptions.digest,
         text: queryOptions.searchText,
-        limit: queryOptions.limit,
```

Comment:
After removing the limit option, the backend will treat it as the default 0 value, then I guess `.Limit(req.Limit)` will make us get 0 rows of the result (I am not sure, need to verify), maybe we need to add a judgment in the backend?

Reply:
right, I'm working on that

Reply:
And I found that the error notification is not intuitive
![image](https://user-images.githubusercontent.com/5772358/98886343-d62bf500-24ce-11eb-9af2-a01dc3d088b9.png)